workers/tests/unittests/test_definitions.py

```diff
@@ -55,3 +64,46 @@ def test_get_fid_fields_api(self):
         self.assertIsInstance(fields, list)
         self.assertIn("name", fields)
         self.assertIn("ogc_fid", fields)
+
+    def test_get_raster_dataset(self):
+        raster = definitions.get_raster_dataset("GHS_BUILT_R2018A")
+        self.assertIsInstance(raster, definitions.RasterDataset)
+
+    def test_get_raster_dataset_undefined(self):
+        with self.assertRaises(RasterDatasetUndefinedError):
+            definitions.get_raster_dataset("foo")
+
+    @mock.patch.dict("os.environ", {}, clear=True)
+    def test_get_data_dir_unset_env(self):
+        data_dir = definitions.get_data_dir()
+        expected = os.path.join(
+            os.path.dirname(
+                os.path.abspath(__file__),
+            ),
+            "..",
```

Comment:
this looks interesting. but probably there is no easier way to write this.

Reply:
It is not very elegant. But I do not know of another way.